_pages/Basic Decarbonization Accounting.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Is it possible to do these without spaces in the names? It'd be a little easier to work with on the CLI. Not a biggie though.

Reply:
yea I can get rid of the spaces. I was following the name for the solar one but now that there are permalinks the filename doesnt matter as much